WeakAuras/WeakAuras.lua

```diff
@@ -5554,3 +5572,23 @@ function WeakAuras.FindUnusedId(prefix)
   end
   return id
 end
+
+function WeakAuras.SerializeTable(t)
```

Comment:
This would be the 4th serialization of tables that we have in WeakAuras, by my count, not including AceSerialize. Do we really need another one?

Reply:
```
local Serializer = LibStub:GetLibrary("AceSerializer-3.0")
print(Serializer:Serialize({a = true, b = true}))
```
> ^1^T^Sa^B^Sb^t^^

```
print(WeakAuras.SerializeTable({a = true, b = true}))
```
> {a=true,b=true,}

Do you which function i could use for this case ?

Reply:
Ah, I suppose AceSerialize is intended for creating strings that can be deserialized with AceSerialize.

I was actually thinking about the DisplayToTableString method in transmission specifically. It might make sense to go ahead and merge that function with this one, since they do functionally the same thing.

Reply:
fixed